package.json

```diff
@@ -10,20 +10,25 @@
     "start:dev": "nodemon -r esm index"
   },
   "devDependencies": {
+    "babel-eslint": "^10.0.1",
     "eslint": "^5.16.0",
     "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-config-prettier": "^4.3.0",
+    "eslint-plugin-flowtype": "^3.9.1",
     "eslint-plugin-import": "^2.17.3",
-    "eslint-plugin-prettier": "^3.1.0",
+    "eslint-plugin-jsx-a11y": "^6.2.1",
+    "eslint-plugin-react": "^7.13.0",
     "nodemon": "^1.19.1"
   },
   "dependencies": {
+    "@hapi/joi": "^15.0.3",
     "dotenv": "^8.0.0",
+    "eslint-config-prettier": "^4.3.0",
+    "eslint-plugin-prettier": "^3.1.0",
     "esm": "^3.2.25",
     "express": "^4.17.1",
     "fancy-log": "^1.3.3",
     "http-status": "^1.3.2",
-    "joi": "^14.3.1",
-    "morgan": "^1.9.1"
+    "morgan": "^1.9.1",
+    "prettier": "^1.17.1"
```

Comment:
prettier should be in `devDepencies ` too

Reply:
Cool let me change